cpp/include/raft/core/bitmap.cuh

```diff
@@ -16,7 +16,7 @@
 
 #pragma once
 
-#include <raft/core/bitset.cuh>
+#include <raft/core/bitmap.hpp>
```

Comment:
Hi, @cjnolet @benfred , inspired by the PR of @lowener https://github.com/rapidsai/raft/pull/2295, I split the bitmap to avoid the compilation issue caused by `*.hpp` files. If we have a better method to resolve this, please let me know; many thanks!

Reply:
Sounds great! Just make sure to keep all cuda stuff out of the hpp file.

Reply:
`bitmap.cuh` should include `bitset.cuh`  instead of the `.hpp` version since it is inheriting from the bitset class. This will avoid having undefined references for `n_elements()` for example

Reply:
> `bitmap.cuh` should include `bitset.cuh` since it is inheriting from the bitset class. This will avoid having undefined references for `n_elements()` for example

Thank you @lowener , very make sense, just fix it!